### ISSUE TITLE
Multiple references

### ISSUE THREAD

**Issue by MarekMatejak** [impact/search#33]
Complex, ObsoleteModelica3, Modelica and ModelicaServices are refering to the same library.

**Comment by dietmarw** [comment]
Yes that is a problem. Technical speaking they all belong to the *meta-package* "Modelica Standard Library" which in its current shape is not really impact compatible and hence appear as just one library. This could be resolved if we could convince the MA to go over to a modularised version of the MSL. Which is still something we are working on.

**Comment by xogeny** [comment]
To elaborate a bit more, when we search we include the repository description.  If multiple libraries exist within the same repository, they all get the same description so any search that matches the repository description will return all libraries in that repository as matches.
